exec_test.go

```diff
@@ -35,3 +36,36 @@ func ExampleCommandExists() {
 	// true
 	// false
 }
+
+func BenchmarkCommandExists(b *testing.B) {
+	cases := []struct {
+		Command string
+	}{
+		{"go"},
+		{"gofmt"},
+		{"foo"},
+		{"bar"},
```

Comment:
still not good

Reply:
@moul There are two commands which always will be in path + 2dummies?

Reply:
why are you checking twice a command which is always in the path
and twice dummies?

foo, bar won't be always dummies, they are too common, please use something more unique

Reply:
look at the README.md generated by you:

```
    BenchmarkCommandExists/go-8                      422287      2896.00 ns/op
    BenchmarkCommandExists/go-parallel-8             307030      3639.00 ns/op
    BenchmarkCommandExists/gofmt-8                   425854      2762.00 ns/op
    BenchmarkCommandExists/gofmt-parallel-8          314979      3795.00 ns/op
    BenchmarkCommandExists/foo-8                      69224     17332.00 ns/op
    BenchmarkCommandExists/foo-parallel-8             86791     14672.00 ns/op
    BenchmarkCommandExists/bar-8                      66331     17619.00 ns/op
    BenchmarkCommandExists/bar-parallel-8             81376     14422.00 ns/op
```

what makes gofmt something interesting to bench just after go, and same question for bar after foo?